packages/core/src/2d/assembler/TiledSpriteAssembler.ts

```
@@ -83,35 +68,35 @@
+    const rowLength = rPos.length - 1;
+    const columnLength = cPos.length - 1;
 
     const subChunk = renderer._subChunk;
     const vertices = subChunk.chunk.vertices;
     const indices = subChunk.indices;
     let count = 0;
     let trianglesOffset = 0;
     for (let j = 0, o = subChunk.vertexArea.start; j < columnLength; j++) {
-      const doubleJ = 2 * j;
+      const doubleJ = j << 1;
+      if (isNaN(cUV.get(doubleJ)) || isNaN(cUV.get(doubleJ + 1))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)